ansible/roles/vm_set/tasks/add_topo.yml

```diff
@@ -185,6 +185,11 @@
   - include_tasks: add_ceos_list.yml
     when: vm_type is defined and vm_type == "ceos"
 
+  - name: Set VM_targets to empty string for ptf topologies
+    set_fact:
+      VM_targets: ""
+    when: topo | search("ptf")
```

Comment:
when: 'ptf' in topo

Reply:
@neethajohn do you have any concerns about using search or some advantage? Thank you.

Reply:
came across threads that mentioned a deprecation warning for this usage. Hence the suggestion